stable/aws-for-fluent-bit/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: aws-for-fluent-bit
 description: A Helm chart to deploy aws-for-fluent-bit project
-version: 0.1.2
-appVersion: 2.2.0
+version: 0.2.0
+appVersion: 2.3.0
```

Comment:
Doesn't the image tag need to be updated as well for this change? https://github.com/aws/eks-charts/blob/master/stable/aws-for-fluent-bit/values.yaml#L7